### DISCUSSION TITLE
How can i display the pyshark live traffic in a new window?

### DISCUSSION THREAD

**Discussion by zgh0stxx** [TomSchimansky/CustomTkinter#1401] [Q&A]
hi, im working on a traffic capturing and analysis tool and im using customTkinter, and I need to display the traffic organized in columns and rows, i need to know how to display an output of pyshark which does not have a gui, this is an example: https://www.youtube.com/watch?v=gstHeldo61w&t
its my graduation project so please if anyone can help, it would make me happy, thanks

**Comment by Akascape** [comment]
@zgh0stxx 
Use matplotlib to plot the graph from the data given by pyshark. You can watch a tutorial on how to use matplotlib.
Matplotlib graph can be placed inside customtkinter frame, just like normal tkinter. 
You can also customize matplotlib colors to match the theme.

**Comment by avalon60** [comment]
You might also want to look at the tkinter Treeview widget. I'm not sure  what the styling options are (to make it blend with Customtkinter), but assuming they exist, and if you want to gild the lily, there are methods for interrogating colour codes, used by the current CustomTkinter theme/appearance mode, which you can then apply to basic Tkinter widgets.

Treeview:
https://youtu.be/YTqDYmfccQU
